saleor/channel/models.py

```diff
@@ -11,6 +12,11 @@ class Channel(models.Model):
     slug = models.SlugField(max_length=255, unique=True)
     currency_code = models.CharField(max_length=settings.DEFAULT_CURRENCY_CODE_LENGTH)
     default_country = CountryField()
+    allocation_strategy = models.CharField(
+        max_length=255,
+        choices=AllocationStrategy.CHOICES,
+        default=AllocationStrategy.PRIORITIZE_SORTING_ORDER,
```

Comment:
Currently we have `PRIORITIZE_HIGH_STOCK` so shouldn't that be the default?

Reply:
For the existing channels it will be `PRIORITIZE_HIGH_STOCK`, for every new channels it will be `PRIORITIZE_SORTING_ORDER`.

Reply:
Clear 👍